2021q3/wg.adoc

```diff
@@ -0,0 +1,19 @@
+## WireGuard on FreeBSD Stabilizes, Eyes Upstreaming
+
+Links +
+link:https://www.wireguard.com/[WireGuard] URL:link:https://www.wireguard.com/[https://www.wireguard.com/] +
+link:https://git.zx2c4.com/wireguard-freebsd/[wireguard-freebsd codebase] URL:link:https://git.zx2c4.com/wireguard-freebsd/[https://git.zx2c4.com/wireguard-freebsd/] +
+
+Contact: Jason A. Donenfeld <Jason@zx2c4.com>
+
+WireGuard is an extremely simple yet fast and modern VPN that utilizes state-of-the-art cryptography.
+
+For the last quarter, the https://git.zx2c4.com/wireguard-freebsd/[wireguard-freebsd codebase] has been quite stable and complete. For a while, there were rapid fire releases fixing issues, and a lot of effort was made to track down every bug report on bugs.freebsd.org, IRC, the mailing list, or otherwise. But by now, the reports have dried up, and mostly users come to IRC with questions on usage and integration, the usual types of things associated with a stabler project. We also have automated CI now for each commit, compiling and running a small smoke test on the repo's supported releases -- 12.1, 12.2, and 13.0. At some point, hopefully that https://git.zx2c4.com/wireguard-freebsd/tree/tests/netns.sh[small smoke test] will expand to include the https://git.zx2c4.com/wireguard-linux/tree/tools/testing/selftests/wireguard/netns.sh[larger battery of tests] from Linux.
```

Comment:
12.1 hasn't been supported since... I think January 2021?

Reply:
12.1 is supported by _the repo_, which in that sentence is clearly referring to the wireguard-freebsd repo. It's not supported by FreeBSD upstream anymore, but people are still running wireguard on old freebsd installs.

Reply:
Instead of using an ambiguous definite article, it can be solved by doing s/the repo\'s/wireguard-freebsd's/.